src/picosvg/svg_reuse.py

```diff
@@ -171,7 +116,7 @@ def _affine_callback(affine, subpath_start, curr_pos, cmd, args, *_unused):
     return ((cmd, args),)
 
 
-def normalize2(shape: SVGShape, tolerance: int = _DEFAULT_TOLERANCE) -> SVGShape:
+def normalize(shape: SVGShape, tolerance: float, ndigits: int) -> SVGShape:
```

Comment:
do we need both a float `tolerance` and a distinct int `ndigits`? Could we derive the latter from the former somehow?
E.g. we could convert the tolerance float to a string and find the position of the decimal point to get the ndigits:`str(tolerance)[::-1].find(".")`. 
str(float) can return exponential notation for small floats, in that case what follows `1e-...` are the ndigits.

Reply:
Maybe I'll explore snap to nearest tolerance

Reply:
Done